GreenXenith/minetest-tools
# Configure JSON schema for `.cdb.json`

Fix https://github.com/GreenXenith/minetest-tools/issues/17

- [x] Configure JSON validation to the schema URL
- [x] Add a local fallback file (will need to be updated from time to time)

**How to test?**

- Open a new `.cdb.json` file
- Press `Ctrl + Space` in the brackets, check if Intellisense activates for possible fields
- Add a license, check if autocompletion works
- Check if hovering a license display "FOSS" if license is FOSS, "NON-FOSS" overwise
- Add some tags, check if autocompletion works
- Check if hovering tags display their description
- Test invalid definition, check if there are warning
- Disconnect from network
- Restart VSCode
- Check autocompletion still works (using the fallback file)
